.github/libs/GithubUtils.ts

```diff
@@ -157,144 +201,142 @@ class GithubUtils {
             isVerified: match[1] === 'x',
         }));
-        return _.sortBy(PRList, 'number');
+        // eslint-disable-next-line no-nested-ternary
+        return PRList.sort((a, b) => (a.number > b.number ? 1 : b.number > a.number ? -1 : 0));
     }
 
     /**
      * Parse DeployBlocker section of the StagingDeployCash issue body.
      *
      * @private
-     *
-     * @param {Object} issue
-     * @returns {Array<Object>} - [{URL: String, number: Number, isResolved: Boolean}]
      */
-    static getStagingDeployCashDeployBlockers(issue) {
-        let deployBlockerSection = issue.body.match(/Deploy Blockers:\*\*\r?\n((?:-.*\r?\n)+)/) || [];
+    static getStagingDeployCashDeployBlockers(issue: OctokitIssueItem): StagingDeployCashBlocker[] {
+        let deployBlockerSection: RegExpMatchArray | null | [] | string = issue.body?.match(/Deploy Blockers:\*\*\r?\n((?:-.*\r?\n)+)/) ?? [];
         if (deployBlockerSection.length !== 2) {
             return [];
         }
+
         deployBlockerSection = deployBlockerSection[1];
-        const deployBlockers = _.map([...deployBlockerSection.matchAll(new RegExp(`- \\[([ x])]\\s(${ISSUE_OR_PULL_REQUEST_REGEX.source})`, 'g'))], (match) => ({
+        const deployBlockers = [...deployBlockerSection.matchAll(new RegExp(`- \\[([ x])]\\s(${ISSUE_OR_PULL_REQUEST_REGEX.source})`, 'g'))].map((match) => ({
             url: match[2],
             number: Number.parseInt(match[3], 10),
             isResolved: match[1] === 'x',
         }));
-        return _.sortBy(deployBlockers, 'number');
+
+        return deployBlockers.sort((a, b) => a.number - b.number);
     }
 
     /**
      * Parse InternalQA section of the StagingDeployCash issue body.
      *
      * @private
-     *
-     * @param {Object} issue
-     * @returns {Array<Object>} - [{URL: String, number: Number, isResolved: Boolean}]
      */
-    static getStagingDeployCashInternalQA(issue) {
-        let internalQASection = issue.body.match(/Internal QA:\*\*\r?\n((?:- \[[ x]].*\r?\n)+)/) || [];
+    static getStagingDeployCashInternalQA(issue: OctokitIssueItem): StagingDeployCashBlocker[] {
+        let internalQASection: RegExpMatchArray | [] | null | string = issue.body?.match(/Internal QA:\*\*\r?\n((?:- \[[ x]].*\r?\n)+)/) ?? [];
         if (internalQASection.length !== 2) {
             return [];
         }
         internalQASection = internalQASection[1];
-        const internalQAPRs = _.map([...internalQASection.matchAll(new RegExp(`- \\[([ x])]\\s(${PULL_REQUEST_REGEX.source})`, 'g'))], (match) => ({
+        const internalQAPRs = [...internalQASection.matchAll(new RegExp(`- \\[([ x])]\\s(${PULL_REQUEST_REGEX.source})`, 'g'))].map((match) => ({
             url: match[2].split('-')[0].trim(),
             number: Number.parseInt(match[3], 10),
             isResolved: match[1] === 'x',
         }));
-        return _.sortBy(internalQAPRs, 'number');
+
+        return internalQAPRs.sort((a, b) => a.number - b.number);
```

Comment:
Same

Reply:
I did the same here, let me know what you think 😄